Makefile

```diff
@@ -354,7 +357,7 @@ openstack_deploy_prep: export KIND=OpenStackControlPlane
 openstack_deploy_prep: openstack_deploy_cleanup $(if $(findstring true,$(NETWORK_ISOLATION)), nmstate nncp netattach metallb metallb_config)  ## prepares the CR to install the service based on the service sample file OPENSTACK
 	$(eval $(call vars,$@,openstack))
 	mkdir -p ${OPERATOR_BASE_DIR} ${OPERATOR_DIR} ${DEPLOY_DIR}
-	pushd ${OPERATOR_BASE_DIR} && git clone -b ${OPENSTACK_BRANCH} ${OPENSTACK_REPO} && popd
+	pushd ${OPERATOR_BASE_DIR} && git clone ${GIT_CLONE_OPTS} $(if $(OPENSTACK_BRANCH),-b ${OPENSTACK_BRANCH}) ${OPENSTACK_REPO} "${OPERATOR_NAME}-operator" && popd
```

Comment:
On doing git clone, the clonned directory is always going to be "${OPERATOR_NAME}-operator". Do we need to it?

Reply:
If we clone from an already cloned local repo, like the one cloned by zuul when using the ci-framework to validate a PR, the clone origin will be path, that may not end with "<operator-name>-operator". I added that at the end to ensure that the operator will be always cloned to the expected dir dispite the clone source.

Reply:
Thank you @pablintino for explaining it, make sense. I would suggest to add an example in the first conversation of this pr. It will be easier for other reviewers.